.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,39 @@
+## Description
 <!-- Briefly describe the change and why it was required. -->
+<!-- If this is a breaking change, explain why it is necessary. Breaking changes must append `!` after the type/scope. See https://ddtrace.readthedocs.io/en/stable/contributing.html for more details. -->
+
+<!-- Copy and paste the relevant snippet based on the type of pull request -->
```

Comment:
Does this selection not work anymore? Did it not work for PRs?

Reply:
Should we automate this checklist based on the conventional commit type?

Reply:
the selection was only ever available via the github CLI 😕 

they don't have any UI for using/selecting a template